src/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -198,4 +213,18 @@ public int compare(final Unit o1, final Unit o2) {
     return Optional.empty();
   }
 
+  private static void loadMaxUnits(final Unit transport, final List<Unit> canBeTransported,
```

Comment:
This method is called in 2 places so inlining it will cause the calling method to become very large and difficult to deal with. `getAvailableCapacity` is a very light method that needs called at least twice even if you in line it since 2 different transport objects are passed. Without a major change this code isn't easy to make much cleaner. Since I have PRs based on this same class and this code is still better than 99% of the code in the project I'd like to move forward. If you want to spend some time looking at the best way to refactor and give more detail then we can do it in a follow on PR.